virtual_rainforest/models/abiotic/radiation.py

```diff
@@ -0,0 +1,340 @@
+
+    cloudy_transmissivity: float = 0.25
+    """Cloudy transmittivity :cite:p:`Linacre1968`"""
+    transmissivity_coefficient: float = 0.50
+    """Angular coefficient of transmittivity :cite:p:`Linacre1968`"""
+    flux_to_energy: float = 2.04
+    """From flux to energy conversion, umol J-1 :cite:p:`Meek1984`"""
+    stefan_boltzmann_constant: float = 5.67e-8
+    """Stefan-Boltzmann constant W m-2 K-4"""
+    soil_emissivity: float = 0.95
+    """Soil emissivity, default for tropical rainforest"""
+    canopy_emissivity: float = 0.95
+    """Canopy emissivity, default for tropical rainforest"""
+    beer_regression: float = 2.67e-5
+    """Parameter in equation for atmospheric transmissivity based on regression of
+    Beer's radiation extinction function :cite:p:`Allen1996`"""
+    celsius_to_kelvin: float = 273.15
+    """Factor to convert temperature in Celsius to absolute temperature in Kelvin"""
+
+
+class Radiation:
+    """Radiation class.
+
+    This class uses a :class:`~virtual_rainforest.core.data.Data` object to populate
+    and store radiation-related attributes which serve as inputs to other modules.
+    Elevation information from a digital elevation model is used for the topographic
+    adjustment of incoming shortwave radiation. The gross radiation that reaches the top
+    of the canopy is reduced as it penetrates through the canopy (the absorption by
+    individual canopy layers is provided by the plants module). What remains is the net
+    shortwave radiation at the surface (= forest floor), which is an input to the
+    :class:`~virtual_rainforest.models.abiotic.Energy_balance` class. Top of canopy
+    photosynthetic photon flux density (PPFD) is the key input for
+    `virtual_rainforest.models.plants` which calculates photosythesis and GPP.
+    Longwave radiation from individual canopy layers and longwave radiation from soil
+    serve as inputs to the :class:`~virtual_rainforest.models.abiotic.Energy_balance`
+    class.
+
+    PPFD and top-of-canopy radiation could be calculated in the AbioticModel __init__
+    for all timesteps provided in data. Something like a `calculate_radiation_balance`
+    function could then update the `longwave_canopy`,`longwave_soil` and
+    `netsurface_radiation` attributes with each time step. This is currently not fully
+    implemented.
+
+    Creating an instance of this class expects a `data` object that contains the
+    following variables:
+    * elevation: elevation above sea level, [m]
+    * shortwave_in: downward shortwave radiation, [J m-2]
+    * canopy_temperature: canopy temperature of individual layers, [C]
+    * surface_temperature: surface soil temperature, [C]
+    * canopy_absorption: absorption by canopy, [J m-2]
+
+    The `data` object can also optionally provide these variables, but will default to
+    the values given below:
+    * albedo_vis: visible light albedo, default = 0.03
+    * albedo_shortwave: shortwave albedo, default = 0.17
```

Comment:
Why were these defaults chosen?

I'd be inclined to make these values constants in the module (e.g. `DEFAULT_ALBEDO_VIS` etc.) and not refer to the values here explicitly, because inevitably at some point someone will change them and forget to update the docstring.